### ISSUE TITLE
Questions about maintainability

### ISSUE THREAD

**Issue by thapar** [AndreSouzaAbreu/vue-data-table#11]
Hey! Thanks for your project! I always have concerns about integrating libraries that are only maintained by a single person, because they often get abandoned eventually. Can I ask how regular updates to the DataTables.net library will be incorporated into your port? Is there a streamlined process to do this or does it take a lot of manual labor to accomplish?

**Comment by AndreSouzaAbreu** [comment]
Hello!
So, this project (vue-data-table) was inspired by DataTables.net. I created it because I wanted some Vue component similiar to DataTables.net, but **free** of jQuery. I built VueDataTable from **scratch** in such way that would resemble DataTables.net in look and functionality. Therefore, this project won't incorporate updates from DataTables.net
I plan to continue to work on this project for at least this year, fixing bugs and adding some functionalities (for example, I plan to add option to search within a single column, and so on). Even if I stop working on this project, I'll probably accept Pull Requests and incorporate them. Moreover, if I really left it away, I can transfer ownership of the project before that happen
